code/__DEFINES/bloodcult.dm

```diff
@@ -0,0 +1,2 @@
+var/global/blood_target = null

```

Comment:
these aren't defines, you've confused yourself, these should go in _GLOBALVARS (or _globalvars it's called something like that)

__DEFINES is strictly for ```#define``` preprocessor directives (except for a few globalvars that are tied to the defines)